docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -292,3 +292,46 @@ flag:
 $ tctl bots rm example
 $ tctl bots add example --roles=foo,bar,machine-id-db
 ```
+
+## Kubernetes connections are failing with `Unable to connect to the server: x509: certificate signed by unknown authority`
+
+### Symptoms
+
+A self-hosted cluster is connecting Machine ID to Kubernetes clusters with the following errors.  This can happen for non-TLS configured 
+Teleport clusters.
+
+```bash
+E0322 22:53:31.653051    1699 memcache.go:265] couldn't get current server API group list: Get "https://teleport.example.com:443/api?timeout=32s": x509: certificate signed by unknown authority
+```
+
+To confirm the TLS routing mode check the `proxy.tls_routing_enabled` from this command with your proxy address. 
+If the value is `false` then this is a non-TLS routing configuration.
+```bash
+curl https://teleport.example.com:443/webapi/ping | jq
+```
+
+### Explanation
+
+Non-TLS routing configured proxies use specific ports for types of traffic. That requires that a Kubernetes
+connection use its designated port. Currently Machine ID requires the Kubernetes public address is set
+to use the correct port. Otherwise it will use the Proxy web port which can cause these
+type of errors.
+
+### Resolution
+
+The `kube_public_addr` is set within the `proxy_service` by administrators. The proxy
+will require a restart after the configuration is updated.
+
+  ```yaml
+  proxy_service:
+    enabled: true
+    kube_listen_addr: 0.0.0.0:3026
+    kube_public_addr: teleport.example.com:3026
+  ```
+
+Use the configuration listing from the proxy  to confirm the `kube_public_address` is set 
```

Comment:
Again, if using "the" name the type of thing you're specifying with `kube_public address`.

Reply:
Please see update

Reply:
Update does not resolve.

Reply:
Please see latest